### ISSUE TITLE
[DLC] pyload says something is broken but it is not ...

### ISSUE THREAD

**Issue by mbosner** [pyload/pyload#1092]
I added a dlc and pyload downloaded everything but says:

```
failedcheckFile() takes no arguments (1 given) 105.00 MiBUploadedTo
```

as soon as i click on "Restart Link" i get (immediately):

```
skipped 105.00 MiBUploadedTo
```

The log says:

```
28.01.2015 21:48:27 INFO      Added package Superpackage 18 links
28.01.2015 21:48:29 INFO      Download starts: SuperCool.part01.rar
28.01.2015 21:48:30 INFO      UploadedTo: Direct download link detected
28.01.2015 21:48:31 INFO      Download starts: SuperCool.part02.rar
28.01.2015 21:48:32 INFO      UploadedTo: Direct download link detected
28.01.2015 21:48:33 INFO      Download starts: SuperCool.part03.rar
28.01.2015 21:48:34 INFO      UploadedTo: Direct download link detected
28.01.2015 21:48:34 WARNING   Download failed: SuperCool.part01.rar | checkFile() takes no arguments (1 given)
28.01.2015 21:48:34 INFO      UnSkipOnFail: Looking for skipped duplicates of: SuperCool.part01.rar (pid:2)
28.01.2015 21:48:34 INFO      UnSkipOnFail: No duplicates found
28.01.2015 21:48:35 INFO      Download starts: SuperCool.part04.rar
28.01.2015 21:48:36 INFO      UploadedTo: Direct download link detected
28.01.2015 21:48:37 INFO      Download starts: SuperCool.part05.rar
28.01.2015 21:48:38 INFO      UploadedTo: Direct download link detected
28.01.2015 21:48:38 WARNING   Download failed: SuperCool.part02.rar | checkFile() takes no arguments (1 given)
28.01.2015 21:48:38 INFO      UnSkipOnFail: Looking for skipped duplicates of: SuperCool.part02.rar (pid:2)
28.01.2015 21:48:38 INFO      UnSkipOnFail: No duplicates found
28.01.2015 21:48:38 WARNING   Download failed: SuperCool.part04.rar | checkFile() takes no arguments (1 given)
28.01.2015 21:48:38 INFO      UnSkipOnFail: Looking for skipped duplicates of: SuperCool.part04.rar (pid:2)
28.01.2015 21:48:38 INFO      UnSkipOnFail: No duplicates found
28.01.2015 21:48:39 INFO      Download starts: SuperCool.part06.rar
28.01.2015 21:48:41 INFO      Download starts: SuperCool.part07.rar
28.01.2015 21:48:43 WARNING   Download failed: SuperCool.part05.rar | checkFile() takes no arguments (1 given)
28.01.2015 21:48:43 INFO      UnSkipOnFail: Looking for skipped duplicates of: SuperCool.part05.rar (pid:2)
28.01.2015 21:48:43 INFO      UnSkipOnFail: No duplicates found
```

**Comment by Killer1337** [comment]
After this Update (UploadedTo (v0.77 -> v0.78) ) the plugin doesn't download anymore.
Logfile output:
`Download failed: FileXY.rar | File not downloadable for free users`

But this is clearly not the case. Via browser it can be downloaded just fine.

**Comment by mbosner** [comment]
Please create a new issue. That seems to be a different problem.

**Comment by Killer1337** [comment]
Seems like it was already adressed in #1096 and fixed in 7a8b6ed
my bad!
